src/Bridges/Constraint/indicator_sos.jl

```diff
@@ -0,0 +1,183 @@
+            (MOI.ScalarAffineFunction{T}, S),
+           ]
+end
+
+function concrete_bridge_type(::Type{<:IndicatorSOS1Bridge{T}},
+                              ::Type{<:MOI.AbstractVectorFunction},
+                              ::Type{MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, S}}) where {T, S<:Union{MOI.LessThan, MOI.GreaterThan}}
+    return IndicatorSOS1Bridge{T, S, MOI.ConstraintIndex{MOI.SingleVariable, S}}
+end
+
+function concrete_bridge_type(::Type{<:IndicatorSOS1Bridge{T}},
+                              ::Type{<:MOI.AbstractVectorFunction},
+                              ::Type{MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, S}}) where {T, S <: MOI.AbstractScalarSet}
+    return IndicatorSOS1Bridge{T, S, Nothing}
+end
+
+# Attributes, Bridge acting as a model
+
+function MOI.get(::IndicatorSOS1Bridge, ::MOI.NumberOfVariables)
+    return 1
+end
+
+function MOI.get(b::IndicatorSOS1Bridge, ::MOI.ListOfVariableIndices)
+    return [b.w_variable_index]
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC, Nothing}, ::MOI.NumberOfConstraints{MOI.SingleVariable, BC}) where {T, BC}
+    return 0
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC, CI}, ::MOI.NumberOfConstraints{MOI.SingleVariable, BC}) where {T, BC, CI <: MOI.ConstraintIndex{MOI.SingleVariable, BC}}
+    return 1
+end
+
+function MOI.get(::IndicatorSOS1Bridge, ::MOI.NumberOfConstraints{MOI.VectorOfVariables, <:MOI.SOS1})
+    return 1
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC}, ::MOI.NumberOfConstraints{MOI.ScalarAffineFunction{T}, BC}) where {T, BC, CI <: MOI.ConstraintIndex{MOI.SingleVariable, BC}}
+    return 1
+end
+
+function MOI.get(b::IndicatorSOS1Bridge{T, BC, CI}, ::MOI.ListOfConstraintIndices{MOI.SingleVariable, BC}) where {T, BC, CI <: MOI.ConstraintIndex}
+    return [b.bound_constraint_index]
+end
+
+function MOI.get(::IndicatorSOS1Bridge{T, BC, Nothing}, ::MOI.ListOfConstraintIndices{MOI.SingleVariable, BC}) where {T, BC}
+    return MOI.ConstraintIndex{MOI.SingleVariable, BC}[]
+end
+
+function MOI.get(b::IndicatorSOS1Bridge{T}, ::MOI.ListOfConstraintIndices{MOI.VectorOfVariables, <:MOI.SOS1}) where {T}
+    return [b.sos_constraint_index]
+end
+
+function MOI.get(b::IndicatorSOS1Bridge{T, BC}, ::MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{T}, BC}) where {T, BC}
+    return [b.linear_constraint_index]
+end
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::MOI.ConstraintPrimalStart,
+    ::Type{<:IndicatorSOS1Bridge})
+    return true
+end
+
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintPrimal, bridge::IndicatorSOS1Bridge)
+    zvalue = MOI.get(model, MOI.VariablePrimal(), bridge.z_variable_index)
+    wvalue = MOI.get(model, MOI.VariablePrimal(), bridge.w_variable_index)
+    lin_primal_start = MOI.get(model, MOI.ConstraintPrimal(), bridge.linear_constraint_index)
+    return [zvalue, lin_primal_start - wvalue]
+end
+
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintPrimalStart, bridge::IndicatorSOS1Bridge)
```

Comment:
I would merge this method and the one above into one as it is done in other bridges supporting stating values

Reply:
In the case of `MOI.ConstraintPrimalStart`, it needs to query `MOI.VariablePrimalStart`, for `MOI.ConstraintPrimal`, it queries MOI.VariablePrimal`. Changing dispatch to reflection with `isa` is not gonna reduce complexity

Reply:
Indeed, then you can keep separate methods but add `attr.N` as argument to `VariablePrimal` constructor and use `attr` instead of `MOI.ConstraintPrimal()` otherwise it won't work with multiple results (that are now exposed in JuMP with https://github.com/JuliaOpt/JuMP.jl/pull/2100)

Reply:
Fixed